maxim/logger/__init__.py

```
@@ -1,22 +1,34 @@
-from .components.attachment import FileAttachment, FileDataAttachment, UrlAttachment
-from .components.error import Error, ErrorConfig
-from .components.feedback import Feedback
-from .components.generation import (
+from .components import (
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)